### PR TITLE
v21.2.15 release notes

### PR DESCRIPTION
**Pull request by gemma-shay**
https://cockroachlabs.atlassian.net/browse/DOC-1902

### CODE REVIEW

File: _includes/releases/v21.2/v21.2.15.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+<h3 id="v21-2-15-bug-fixes">Bug fixes</h3>
+
+- Fixed incorrect error handling that could cause casts to OID types to fail in some cases. [#85125][#85125]
+- Fixed a bug introduced in v20.2 that could cause a panic when an expression contained a geospatial comparison like `~` that was negated. [#84629][#84629]
+- Nodes no longer gossip information about table statistics once all nodes in the cluster are upgraded to v21.2. [#85494][#85494]
+- Fixed an error that could occur when a query included a limited reverse scan and some rows needed to be retrieved by `GET` requests. [#85583][#85583]
+- Fixed a bug where clients could sometimes receive errors due to lease acquisition timeouts of the form `operation "storage.pendingLeaseRequest: requesting lease" timed out after 6s`. [#85429][#85429]
+- Fixed a bug that could cause union queries to return incorrect results in rare cases. [#85653][#85653]
+- Fixed a bug that could cause a panic in rare cases when the unnest() [function](../v21.2/functions-and-operators.html) was used with a tuple return type.  [#85347][#85347]
```
Comment (andf-crl) [inline _includes/releases/v21.2/v21.2.15.md:28]:
```suggestion
- Fixed a bug that could cause a panic in rare cases when the `unnest()` [function](../v21.2/functions-and-operators.html) was used with a `tuple` return type.  [#85347][#85347]
```

File: _includes/releases/v21.2/v21.2.15.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+- [`cockroach debug zip`](../v21.2/cockroach-debug-zip.html) now includes all system tables by default, except for a few (a deny list). [#86142][#86142]
+
+<h3 id="v21-2-15-bug-fixes">Bug fixes</h3>
+
+- Fixed incorrect error handling that could cause casts to OID types to fail in some cases. [#85125][#85125]
+- Fixed a bug introduced in v20.2 that could cause a panic when an expression contained a geospatial comparison like `~` that was negated. [#84629][#84629]
+- Nodes no longer gossip information about table statistics once all nodes in the cluster are upgraded to v21.2. [#85494][#85494]
+- Fixed an error that could occur when a query included a limited reverse scan and some rows needed to be retrieved by `GET` requests. [#85583][#85583]
+- Fixed a bug where clients could sometimes receive errors due to lease acquisition timeouts of the form `operation "storage.pendingLeaseRequest: requesting lease" timed out after 6s`. [#85429][#85429]
+- Fixed a bug that could cause union queries to return incorrect results in rare cases. [#85653][#85653]
```
Comment (andf-crl) [inline _includes/releases/v21.2/v21.2.15.md:27]: - Could link `union` to here: `selection-queries.html#union-combine-two-queries`

File: _includes/releases/v21.2/v21.2.15.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+- Removed the `DatabaseID` field from the sampled query telemetry log due to its potential to cause indefinite blocking in the case of a lease acquisition failure. [#85022][#85022]
+
+<h3 id="v21-2-15-operational-changes">Operational changes</h3>
+
+- Telemetry logs now display more finely redacted error messages from SQL execution. Previously, the entire error string was redacted. [#85463][#85463]
+
+<h3 id="v21-2-15-command-line-changes">Command-line changes</h3>
+
+- The CLI now contains a flag (`--log-config-vars`) that allows for environment variables to be specified for expansion within the logging configuration file. This allows a single logging configuration file to service an array of sinks without further manipulation of the configuration file. [#85172][#85172]
+- [`cockroach debug zip`](../v21.2/cockroach-debug-zip.html) now includes all system tables by default, except for a few (a deny list). [#86142][#86142]
```
Comment (andf-crl) [inline _includes/releases/v21.2/v21.2.15.md:18]: Betting I'll need to track down related changes to https://www.cockroachlabs.com/docs/v21.2/cockroach-debug-zip.html#files then. I'll handle it.

File: _includes/releases/v21.2/v21.2.15.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v21-2-15-sql-language-changes">SQL language changes</h3>
+
+- Removed the `DatabaseID` field from the sampled query telemetry log due to its potential to cause indefinite blocking in the case of a lease acquisition failure. [#85022][#85022]
+
+<h3 id="v21-2-15-operational-changes">Operational changes</h3>
+
+- Telemetry logs now display more finely redacted error messages from SQL execution. Previously, the entire error string was redacted. [#85463][#85463]
```
Comment (andf-crl) [inline _includes/releases/v21.2/v21.2.15.md:13]: For lines 7 - 13: I have been instructed in the past to remove sampled query / telemetry change log entries from these RNs. I think someone would want this removed here also, but I don't know for sure.

**Review by andf-crl — APPROVED** [review]
LGTM with some small suggestions. Thanks!

**Comment by gemma-shay** [comment]
tftr!

**Pull request merged**